modules/dnn/src/onnx/opencv-onnx.proto

```diff
@@ -0,0 +1,446 @@
+//
+// WARNING: This file is automatically generated!  Please edit onnx.in.proto.
+//
+
+
+// Copyright (c) Facebook Inc. and Microsoft Corporation.
+// Licensed under the MIT license.
+
+syntax = "proto2";
+
+package onnx;
```

Comment:
To avoid conflicts usually we need to rename package name too (see "opencv_caffe", #10139):
- something like `opencv_onnx`

Reply:
done

Reply:
Need to re-generate "pb" files too.